build/RsyncPlugin.js

```diff
@@ -10,7 +10,7 @@ RsyncPlugin.prototype.apply = function rsync(compiler) {
     compiler.plugin('done', () => {
         console.log('');
         console.log(`🔄 🔄 🔄  Rsync starting for ${this.source} 🔄 🔄 🔄`);
-        execSync(`rsync -avz --delete --exclude=".*" "${this.source}" "${this.destination}"`, { stdio: [0, 1, 2] });
```

Comment:
Note that this can potentially result in a lot of cruft if you change file names around or do some larger configuration change.